ui/components/ui/identicon/identicon.stories.js

```diff
@@ -1,47 +1,64 @@
 import React from 'react';
-import { text, boolean, number } from '@storybook/addon-knobs';
+import README from './README.mdx';
 import Identicon from './identicon.component';
 
 export default {
-  title: 'Identicon',
+  title: 'Components/UI/Identicon',
   id: __filename,
+  component: Identicon,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    addBorder: { control: 'boolean' },
+    address: { control: 'text' },
+    className: { control: 'text' },
+    diameter: { control: 'number' },
+    image: { control: 'text' },
+    useBlockie: { control: 'boolean' },
+    alt: { control: 'boolean' },
+    imageBorder: { control: 'boolean' },
+    useTokenDetection: { control: 'boolean' },
+    tokenList: { control: 'object' },
+  },
 };
 
-const diameterOptions = {
-  range: true,
-  min: 10,
-  max: 200,
-  step: 1,
+export const DefaultStory = (args) => <Identicon {...args} />;
+
+DefaultStory.storyName = 'Default';
+DefaultStory.args = {
+  addBorder: false,
+  address: '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1',
+  diameter: 46,
+  useBlockie: false,
 };
-export const standard = () => (
-  <Identicon
-    addBorder={boolean('Add Border', Identicon.defaultProps.addBorder)}
-    address={text('Address', '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1')}
-    diameter={number(
-      'Diameter',
-      Identicon.defaultProps.diameter,
-      diameterOptions,
-    )}
-    useBlockie={boolean('Use Blockie', Identicon.defaultProps.useBlockie)}
-  />
-);
 
-export const image = () => <Identicon image="./images/eth_logo.svg" />;
+export const WithImage = (args) => <Identicon {...args} />;
+WithImage.args = {
+  addBorder: false,
+  address: '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1',
+  diameter: 46,
+  useBlockie: false,
+  image: './images/eth_logo.svg',
+};
 
-export const blockie = () => (
-  <Identicon
-    address={text('Address', '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1')}
-    useBlockie={boolean('Use Blockie', true)}
-  />
-);
+export const WithBlockie = (args) => <Identicon {...args} />;
+WithBlockie.args = {
+  addBorder: false,
+  address: '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1',
+  diameter: 46,
+  useBlockie: true,
+};
 
-// The border size is hard-coded in CSS, and was designed with this size identicon in mind
+// // The border size is hard-coded in CSS, and was designed with this size identicon in mind
```

Comment:
We don't need this second set of inline comment slashes.

Reply:
Removed comment and have updated component to use dynamic values for width, height and border radius so `withBorder` prop works at any size